pkgs/applications/misc/k40-whisperer/default.nix

```diff
@@ -41,19 +54,29 @@ in stdenv.mkDerivation rec {
   buildPhase = "";
 
   installPhase = ''
+    runHook preInstall
+
     mkdir -p $out
     cp -p * $out
 
     mkdir -p $out/bin
     mkdir -p $out/lib/udev/rules.d
+    mkdir -p $out/share/icons/hicolor/128x128/apps
+    mkdir -p "$out/share/applications"
```

Comment:
Is this still needed with `copyDesktopItems`?

Reply:
Line 65 wasn't needed, I've removed it